OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -25,27 +25,39 @@
     <Reference Include="Eluant, Version=1.0.5229.27703, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\thirdparty\Eluant.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
I suggest looking into how the other projects handle this.  The specific mod dlls all reference Mods.Common.